frontend/app_flowy/packages/appflowy_editor/lib/src/render/emoji/emoji_select_widget.dart

```diff
@@ -0,0 +1,198 @@
+import 'package:appflowy_editor/src/core/document/node.dart';
```

Comment:
It seems that you didn't merge the origin/main before coding. This file has been moved to package `appflowy_editor_plugin `. So please don't submit it again.